### PR TITLE
BETA: Register sharan.is-a.dev

### PR DESCRIPTION
**Pull request by Vasoyasharan**
Added `sharan.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by triach-rold — CHANGES_REQUESTED** [review]
404. Unreachable.

**Review by MaskDuck — DISMISSED** [review]
eh, this website returns 404?

**Comment by Vasoyasharan** [comment]
> eh, this website returns 404?

Is it reachable now?

**Comment by Vasoyasharan** [comment]
> 404. Unreachable.

Is it reachable now?

**Review by triach-rold — APPROVED** [review]
[This](https://vasoyasharan.github.io/Mywebsite/) website is now reachable and complete. Nice work and good projects.

**Comment by Vasoyasharan** [comment]
> [This](https://vasoyasharan.github.io/Mywebsite/) website is now reachable and complete. Nice work and good projects.

thanks .. but it's domain is giving error..

Screenshot from 2024-05-03 09-00-57

**Comment by CuteDog5695** [comment]
> > [This](https://vasoyasharan.github.io/Mywebsite/) website is now reachable and complete. Nice work and good projects.
> 
> thanks .. but it's domain is giving error..
> 
> Screenshot from 2024-05-03 09-00-57

thats because PR wasnt merged

File: domains/sharan.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Sharan's website",
+  "repo": "https://github.com/Vasoyasharan/Mywebsite",
+  "owner": {
+    "username": "Vasoyasharan",
+    "email": "sharanvasoya@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://vasoyasharan.github.io/Mywebsite/"
```
Comment (CuteDog5695) [inline domains/sharan.json:9]: remove https:// and /mywebsite/ from CNAME

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
seems fine now

File: domains/sharan.json (hunk 1 above)
Comment (Vasoyasharan) [reply domains/sharan.json]: thanks..

**Pull request merged**